backend/src/models/db.js

```diff
@@ -0,0 +1,93 @@
+const sqlite3 = require('sqlite3')
+const db = new sqlite3.Database('./db/questions.db')
+
+/* Used a tutorial to help get started...
+ * This will be changed in the very near future.
```

Comment:
I'll leave as is then for now. One thing is probs better to not tie the db startup to the server in case you need to restart the server etc. Although probably fine for development in short term.